packages/browser/README.md

```diff
@@ -19,3 +19,18 @@ document.body?.addEventListener('click', () => {
   analytics.track('document body clicked!')
 })
 ```
+
+## Other Regions
+
+If you're using a [different data center](https://customer.io/docs/accounts-and-workspaces/data-centers/) such as our EU region, you can specify an alternate endpoint:
```

Comment:
Not serious, but I don't think we have plans to introduce alternate data centers. I'd just make this more "EU" focused. 

If you're in our EU region, you'll need to specify our regional endpoint.